content/blog/2016/2016-08-29-jenkins-world-speaker-blog-goodgame.adoc

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Enforcing Jenkins Best Practices"
+tags:
+- event
+- JenkinsWorld
+author: hinman55
+---
+
+NOTE: This is a guest post by Jenkins World speaker David Hinske, Release Engineer at link:https://www.goodgamestudios.com/[Goodgame Studios].
+
+image:/images/post-images/Jenkins-World-logo.png [role=right]
+
+Hey there, my name is David Hinske and I work at Goodgame Studios (GGS), a game development company in Hamburg, Germany. As Release Engineer in a company with several development teams, it comes in handy using several Jenkins-instances. While this approach works fine in our company and gives the developers a lot of freedom, we came across some long-term problems concerning maintenance and standards. Problems, which where mostly caused by misconfiguration/non-usage of plugins. With “configuration as code” in mind, I took the approach to apply static code-analysis with the help of SonarQube, a platform to manage code quality, for all of our Jenkins-Job-configuration.
```

Comment:
Some unnecessary use of hyphens:
"Jenkins instances"
"code analysis"
"Jenkins job configurations"

Also, the fourth sentence is a fragment. I would write, "These problems were mostly caused by misconfiguration or non-use of plugins."